src/official-account/official-account.ts

```diff
@@ -1,36 +1,40 @@
 /* eslint-disable camelcase */
 import { EventEmitter } from 'events'
+import { v4 } from 'uuid'
 
 import crypto from 'crypto'
-import { ContactGender, FileBox, log }  from 'wechaty-puppet'
+import { ContactGender, FileBox, log } from 'wechaty-puppet'
 import { FileBoxType } from 'file-box'
 
 import { normalizeFileBox } from './normalize-file-box'
 
 import {
   Webhook,
   VerifyArgs,
-}                      from './webhook'
+} from './webhook'
 import { PayloadStore } from './payload-store'
 import {
   getSimpleUnirest,
   SimpleUnirest,
-}                       from './simple-unirest'
+} from './simple-unirest'
 import {
   OAMessageType,
   // OAMediaPayload,
   OAMediaType,
   ErrorPayload,
   OAContactPayload,
-}                       from './schema'
+  OATagPayload,
+  OAMessagePayload,
+} from './schema'
+import { getTimeStampString } from './utils'
 
 export interface OfficialAccountOptions {
-  appId            : string,
-  appSecret        : string,
-  port?            : number,
-  token            : string,
-  webhookProxyUrl? : string,
-  personalMode?    : boolean,
+  appId             : string,
+  appSecret         : string,
+  port?             : number,
+  token             : string,
+  webhookProxyUrl?  : string,
+  personalMode?     : boolean,
```

Comment:
Please remove the unrelated one more space in the file, so that those lines could be kept no change at all in the PR.

Reply:
```typescript
export interface OfficialAccountOptions {
  appId           : string,
  appSecret       : string,
  port?           : number,
  token           : string,
  webhookProxyUrl?: string,
  personalMode?   : boolean,
}
```
the indent in `ts` is 2, and the length of `webhookProxyUrl?` is 16. So there should be no indent between `webhookProxyUrl?` and `:`. Is that right ?

Or, is there  always a blank indent after `?` symbol ?

Reply:
If you use the VS Code plugin `Better Align` to align them, then we can get a consistent result: always add one space without care about any indent size.

Please try the Better Align plugin and then I believe everything will be ok because we have saved its setting in our `.vscode/` config files.